docs/toc.js

```diff
@@ -490,11 +490,6 @@ module.exports = {
               title: 'Component Story Format',
               type: 'link',
             },
-            {
```

Comment:
This page was removed because it was almost entirely redundant with an existing page.

Redirect to `/docs/writing-docs/mdx` added in https://github.com/storybookjs/frontpage/pull/451.